flux-cli/src/main/java/com/marklogic/flux/impl/importdata/SplitterParams.java

```diff
@@ -129,6 +129,42 @@ public class SplitterParams implements SplitterOptions {
     )
     private String xmlNamespace;
 
+    @CommandLine.Option(
+        names = "--splitter-classifier-host",
```

Comment:
Just do `--classifier-` to start with, as we know we don't want the classifier to require the splitter.